README.md

```diff
@@ -111,3 +113,6 @@ allIdentifiers.getKlaviyoId(); // == 777
 attentiveConfig.clearUser();
 // When/if a user logs back in, `identify` should be called again with the logged in user's identfiers
 ```
+
+## Minimum Version Support
+The Attentive Android SDK currently supports Android API Level 26 and above. The SDK will still build on versions below 26, but functionality will no-op.
```

Comment:
Does this mean our SDK won't work on any versions below 26?

Reply:
below api level 26 may not have full support - some features may still work while others don't